### ISSUE TITLE
Boost Graph Library?

### ISSUE THREAD

**Issue by Ulysses1337** [awulkiew/graphical-debugging#37]
It's a bit different from the visuals available today, but it would be quite useful to have. BGL graphs can be exported to Graphviz, which can be fed into an available layout engine.

https://www.boost.org/doc/libs/1_76_0/libs/graph/doc/index.html
https://graphviz.org

**Comment by awulkiew** [comment]
Yes, I think this is a good idea. It'd probably require adding GraphWatch. I'd prefer to calculate layout in the extension itself, maybe there are libraries for that already. But unfortunately I don't have time for this and many things are in the queue already.
